airflow/models/taskinstance.py

```diff
@@ -1522,7 +1523,7 @@ def is_eligible_to_retry(self):
         return self.task.retries and self.try_number <= self.max_tries
 
     @provide_session
-    def get_template_context(self, session=None) -> Context:
+    def get_template_context(self, ignore_param_exceptions=True, session=None) -> Context:
```

Comment:
Why is the default to ignore exceptions?

Reply:
`get_template_context` is being called from many places, but we want to raise exceptions from a couple of places only, hence keeping it to ignore by default.